apps/test/unit/templates/referenceGuides/ReferenceGuidesViewTest.js

```diff
@@ -20,13 +27,77 @@ describe('ReferenceGuideView', () => {
   it('reference guide shows content', () => {
     const referenceGuide = {
       display_name: 'display name',
-      content: 'markdown text'
+      content: 'markdown text',
+      position: 0,
+      key: 'guide',
+      parent_reference_guide_key: 'concepts'
     };
     const wrapper = isolateComponent(
-      <ReferenceGuideView referenceGuide={referenceGuide} />
+      <ReferenceGuideView
+        referenceGuide={referenceGuide}
+        referenceGuides={[referenceGuide]}
+      />
     );
     expect(wrapper.findOne('EnhancedSafeMarkdown').props.markdown).to.equal(
       referenceGuide.content
     );
   });
+
+  it('reference guide formats categories for nav bar', () => {
+    const referenceGuide = {
+      display_name: 'display name 3',
+      content: 'content 3',
+      key: 'guide3',
+      position: 0,
+      parent_reference_guide_key: 'guide1'
+    };
+    const referenceGuides = [
```

Comment:
Since we have a bunch of 3 level nesting in the actual reference guides it would probably be good to have that here.